docs/plugins/metrics.md

```diff
@@ -221,28 +221,39 @@ The `Metrics.user` namespace is used for tracking media events.
 
 The [VideoPlayer plugin](/plugins/videoplayer) automatically tracks and sends Metrcs for the following media related events:
 
+- LoadStart
 - Abort
 - CanPlay
 - Ended
 - Pause
 - Play
+- Progress
 - Suspend
 - VolumeChange
 - Waiting
 - Seeking
 - Seeked
 - Ratechange
+- Rendition
 
 For apps that don't use the VideoPlayer plugin, tracking has to be implemented manually. Media metrics are initiated by
-calling the method `Metrics.media()` and passing it the url of the media asset that is being tracked.
+calling the method `Metrics.media()` and passing it the uri of the media asset that is being tracked. The uri may be a qualified URL to the CDN, or a unique identifier from a CMS.
```

Comment:
Yes, this is in line with what we discussed today. From an SDK perspective this will work fine (i.e. either a full url or an assetId as a grouping value).

@wouterlucas do you see any impact on the side of the stats server?

Reply:
Works for me.